### PR TITLE
📝 docs(release): Update baize release docs

### PR DESCRIPTION
**Pull request by samzong**
This pull request updates the release notes in both English and Chinese documentation to include the latest features and improvements for versions v0.9.0 and v0.8.0 of AI Lab.

#### English Documentation (`docs/en/docs/baize/intro/release-notes.md`):

* **v0.9.0 (2024-09-30)**:
  - Added a new data management sub-module “Data Annotation” for managing data annotation capabilities for mainstream data types.
  - Added a new model management sub-module “Model List” supporting quick data import.
  - Added the ability to specify PVC storage space size when creating a “Dataet”.
  - Added support for one-click restart of “Training Job”.
  - Added the ability to specify GPU card type when using vGPU resources.
  - Upgraded the base image of “baize-notebook” to v0.9.0.
  - Improved global alerts while ensuring data availability during cluster exceptions.

* **v0.8.0 (2024-08-31)**:
  - Added support for manually and automatically saving as an image while a `Notebook` is running or closed, respectively.
  - Added support for selecting private images in the image repository for `Notebook` and `Training Job` images.
  - Added support for configuring **data input** and **data output** for `Notebook` and `Training Job`, directly associating with datasets.
  - Added support for configuring `Notebook` to start as `Root`.
  - Added support for configuring breakpoint continuation for `Training Job` and automatically detecting and repairing task failures.
  - Added display of task parameter information in the details of `Training Job`.

### CODE REVIEW

File: docs/en/docs/baize/intro/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,45 @@ date: 2024-07-12
...
+## 2024-09-30
+
+### v0.9.0
+
+#### Features
+
+- **Added** Added a new data management sub-module “Data Annotation” which can manage data annotation capabilities for
+  mainstream data types.
+- **Added** Added a new model management sub-module “Model List” which supports quick data import.
+- **Added** When creating a “Data Set”, the PVC storage space size can be specified.
```
Comment (kebe7jun) [inline docs/en/docs/baize/intro/release-notes.md:20]:
```suggestion
- **Added** When creating a “Dataset”, the PVC storage space size can be specified.
```

File: docs/en/docs/baize/intro/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,45 @@ date: 2024-07-12
...
+- **Added** Supports one-click restart of “training tasks”.
```
Comment (kebe7jun) [inline docs/en/docs/baize/intro/release-notes.md:21]:
```suggestion
- **Added** Supports one-click restart of “Training jobs”.
```

File: docs/en/docs/baize/intro/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,45 @@ date: 2024-07-12
...
+
+## 2024-08-31
+
+### v0.8.0
+
+#### Features
+
+- [Beta] **Added** Support manually saving as an image while a `Notebook` is running (depends on the image repository
+  module).
+- [Beta] **Added** Support automatically saving as an image when a `Notebook` is closed (depends on the image repository
```
Comment (windsonsea) [inline docs/en/docs/baize/intro/release-notes.md:34]:
```suggestion
- **Added** [Beta] support manually saving as an image while a `Notebook` is running (depends on the image repository
  module).
- **Added** [Beta] support automatically saving as an image when a `Notebook` is closed (depends on the image repository
```

**Review by windsonsea — COMMENTED** [review]
提交中文就可以了，英文留给同事，现在机翻的风格跟以前都不一致

File: docs/en/docs/baize/intro/release-notes.md (hunk 3 above)
Comment (samzong) [reply docs/en/docs/baize/intro/release-notes.md:34]: 因为发现上个月的英文没提交，所以顺便补充了。

**Review by windsonsea — APPROVED** [review]

**Pull request merged**